api/src/main/resources/org/openmrs/api/db/hibernate/PatientProgram.hbm.xml

```diff
@@ -11,69 +11,75 @@
 
 -->
 <!DOCTYPE hibernate-mapping PUBLIC
-    "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
-    "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">   
+        "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
 <hibernate-mapping package="org.openmrs">
 
-<class name="org.openmrs.PatientProgram" table="patient_program">
-
-	<id name="patientProgramId" type="java.lang.Integer" column="patient_program_id">
```

Comment:
Can you reduce the scope of changes in this file by only touching what changed? Do not move around anything because then they will appear to have changed.